src/lib/Theme.ts

```diff
@@ -952,7 +956,29 @@ export default class Theme {
             theme.config = theme.config || {};
             theme.config.global_schema = globalConfigSchema;
             theme.config.current = globalConfigData.current || 'default';
-            theme.config.list = globalConfigData.list || [{ name: 'default' }];
+
+            // Modify list to update deleted page's prop
+            let newList = null;
```

Comment:
shouldn't this be ?

```
let newList = globalConfigData.list;
```

Reply:
Doesn't matters, check line 963, contents of `newList` is mapped to new filtered data from `globalConfigData.list`

Reply:
`allowedDefaultProps` will only have keys if there are any system pages to delete. What if #962 condition is not met?

Reply:
Agreed. So 2 change.
- `theme.config.list = newList`
- let newList = globalConfigData.list;